src/NativeScript/SymbolLoader.mm

```diff
@@ -6,75 +6,146 @@
 //  Copyright (c) 2014 г. Telerik. All rights reserved.
```

Comment:
You could also rename the header from `SymbolLoader.cpp` to `SymbolLoader.mm` if you want :D